docs/12-self-hosting/03-host-creation/03-QEMU/05-macos.mdx

```diff
@@ -0,0 +1,280 @@
+    -drive id=MacHDD,if=none,cache=none,format=qcow2,aio=threads,file="mac_hdd_ng.img" \
+    -serial stdio -vga virtio -parallel none \
+    -device virtio-net-pci,netdev=network \
+    -usbdevice tablet \
+    -device usb-kbd,bus=ehci.0 \
+    -netdev user,id=network,hostfwd=tcp::"${VM_SSH_PORT}"-:"${HOST_SSH_PORT}" \
+    -vnc $HOST_ADDRESS:$VNC_PORT
+  ```
+
+## Connect to the VM using VNC.
+
+- In your VNC software use the address format `$HOST_ADDRESS:$VNC_PORT` to connect to the VM.
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={Vnc} width="600" />
+  </div>
+  <br />
+  <br />
+
+## Format Virtual Drive
+
+- Select install media
+
+  Choose the 'macOS Base System' option using arrow-keys to select and enter to confirm.
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={InstallMedia} width="600" />
+  </div>
+  <br />
+  <br />
+
+- Enter the disk utility
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={DiskUtility} width="600" />
+  </div>
+  <br />
+  <br />
+
+- Format and rename the empty storage volumes
+
+  The disk should show up as `Apple Inc. VirtIO Block Media` in your Disk Utility window. Select the
+  `Erase` tool to reformat the volumes and change the name to `Macintosh HD`.
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={FormatDisk} width="600" />
+  </div>
+  <br />
+  <br />
+
+- Quit the disk utility
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={QuitDisk} width="600" />
+  </div>
+  <br />
+  <br />
+
+## Install MacOS
+
+- Choose the `Reinstall MacOS` option from the main menu
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={Reinstall} width="600" />
+  </div>
+  <br />
+  <br />
+
+- Choose your formatted volumes as the install location
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={InstallLocation} width="600" />
+  </div>
+  <br />
+  <br />
+
+- Monitor the install process
+
+  The VM will reboot several times during the installation process. After the first reboot, you will
+  have a new option named 'MacOS Installer'. Select this option as your boot device.
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={BootMedia} width="600" />
+  </div>
+  <br />
+  <br />
+
+  When the first stage of the installation has completed, the 'MacOS Installer' option will change
+  to 'Macintosh HD' (or whatever you named your drive). Continue to select this option as the boot
+  device for all subsequent reboots.
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={BootMedia2} width="600" />
+  </div>
+  <br />
+  <br />
+
+- Boot into MacOS and complete the account creation process
+
+  Finally, your VM should boot into the MacOS user setup screen and allow you to create your
+  account.
+
+  <div style={{ textAlign: 'center' }}>
+    <img src={CreateAccount} width="600" />
+  </div>
+  <br />
+  <br />
+
+## Enable Remote Login
+
+- Enable 'Remote Login' from the 'Sharing' menu in System Settings
```

Comment:
We should probably mention something about security considerations here. When enabling remote login or SSH, there are security risks involved.

Reply:
added to macos section in commit https://github.com/game-ci/documentation/pull/419/commits/bdcf1f748784adbaa2709b1f07e98b58c11c79c4